### PR TITLE
feat(molgenis-vm): Service for node-exporter

### PR DESCRIPTION
**Pull request by fdlk**
Relevant scrape config:
```
- job_name: kubernetes-services
  kubernetes_sd_configs:
  - role: service
  metrics_path: /metrics
  relabel_configs:
    - source_labels: [__meta_kubernetes_service_annotation_prometheus_io_scrape]
      action: keep
      regex: true
    - source_labels: [__meta_kubernetes_service_annotation_prometheus_io_path]
      action: replace
      target_label: __metrics_path__
      regex: (.+)
    - source_labels:
      - __meta_kubernetes_service_label_molgenis_org_environment
      action: replace
      target_label: environment
      regex: (.+)
```

#### Checklist
- [ ] Functionality works & meets specifications (tested on rancher)
- [ ] Templates reviewed
- [ ] Added values to questions
- [ ] Bumped the chart version
- [ ] Updated appVersion (if needed)

### CODE REVIEW

File: charts/molgenis-vm/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A chart to expose services on a MOLGENIS VM
+home: https://github.com/prometheus/node_exporter
+icon: https://img.icons8.com/ios-filled/50/000000/vmware.png
+name: molgenis-vm
```
Comment (sidohaakma) [inline charts/molgenis-vm/Chart.yaml]: I would rename it to something more descriptive.

File: charts/molgenis-vm/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+appVersion: "1.0"
```
Comment (sidohaakma) [inline charts/molgenis-vm/Chart.yaml]: Maybe couple it to relevant MOLGENIS version?

File: charts/molgenis-vm/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+description: A chart to expose services on a MOLGENIS VM
```
Comment (sidohaakma) [inline charts/molgenis-vm/Chart.yaml]: Is this not referring to exposing the JVM metrics of a MOLGENIS instance?

**Comment by sidohaakma** [comment]
Linter fails ;-D.

File: charts/molgenis-vm/Chart.yaml (hunk 3 above)
Comment (fdlk) [reply charts/molgenis-vm/Chart.yaml]: Right now it does the nodeexporter for the VM

**Comment by sidohaakma** [comment]
Maybe later!

**Pull request closed without merging**